source/acuity-metadata.json

```diff
@@ -1,70 +1,235 @@
 [
   {
     "name": "a_times_b_plus_c",
+    "category": "Layer",
+    "attributes": [],
     "inputs": [
-      { "name": "A" },
-      { "name": "B" },
-      { "name": "C" }
+      { "name": "input0" },
+      { "name": "input1" },
+      { "name": "input2" }
     ],
+    "constants": [],
     "outputs": [
       { "name": "output" }
     ]
   },
   {
     "name": "abs",
+    "category": "Layer",
```

Comment:
Follow the category patterns in other files. `Layer` is used Convolution, LSTM or Fully Connected layers.